### DISCUSSION TITLE
[Feat] setZoom method

### DISCUSSION THREAD

**Discussion by lucadalli** [google/model-viewer#2674] [Ideas]
I think a `setZoom` method alongside properties `minZoom` and `maxZoom` which allow zoom to be set programmatically would be a good addition. 

This would enable zoom sliders like this one:
![image](https://user-images.githubusercontent.com/39853447/128620224-e340a2a9-d239-46f3-a348-12dd846a094c.png)

**Comment by elalish** [comment]
We already have this, but it's set by two parameters: camera radius and fieldOfView: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-cameraOrbit and https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-fieldOfView

**Reply by lucadalli** [reply]
Manipulating `camera-radius` as a percentage value seems to be sufficient to achieve the desired zoom in and out effect. Thanks.
